project/Build.scala

```diff
@@ -14,8 +18,9 @@ object Build extends sbt.Build {
     .settings(libraryDependencies ++= Dependencies.avro ++ Dependencies.test)
     .settings(Packaging.settings: _*)
     .settings(sbtavro.SbtAvro.avroSettings ++ avroSettingsTest: _*)
-    .settings(instrumentSettings: _*)
+    //.settings(instrumentSettings: _*)
```

Comment:
This can be safely removed, as if one day we reactivate the `ScoverageSbtPlugin` it does not require this line on version > 0.99.x